inst/extdata/installPackages.R

```diff
@@ -34,18 +60,59 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
   }
 }
 
-lib <- Sys.getenv("R_LIBS")
-if (nchar(lib) == 0) {
-  stop("Could not get a value for R_LIBS environment variable")
+nodename <- Sys.info()["nodename"]
```

Comment:
not sure what this is for? can we divorce the local machine idea from what we're building on the server? this applies to the DOI root cert too.. I think you're mixing scope again.